src/components/CalculatorUI.js

```diff
@@ -0,0 +1,180 @@
+            handleClick(e);
+          }}
+        >
+          +/-
+        </button>
+        <button
+          type="submit"
+          onClick={(e) => {
+            handleClick(e);
+          }}
+        >
+          %
+        </button>
+        <button
+          type="submit"
+          className="side"
+          onClick={(e) => {
+            handleClick(e);
+          }}
+        >
+          ÷
+        </button>
+        <button
+          type="submit"
+          onClick={(e) => {
+            handleClick(e);
+          }}
+        >
+          7
+        </button>
+        <button
+          type="submit"
+          onClick={(e) => {
+            handleClick(e);
+          }}
+        >
+          8
+        </button>
+        <button
+          type="submit"
+          onClick={(e) => {
+            handleClick(e);
+          }}
+        >
+          9
+        </button>
+        <button
+          type="submit"
+          className="side"
+          onClick={(e) => {
+            handleClick(e);
+          }}
+        >
+          x
+        </button>
+        <button
+          type="submit"
+          onClick={(e) => {
+            handleClick(e);
+          }}
+        >
+          4
+        </button>
+        <button
+          type="submit"
+          onClick={(e) => {
+            handleClick(e);
+          }}
+        >
+          5
+        </button>
+        <button
+          type="submit"
+          onClick={(e) => {
+            handleClick(e);
+          }}
+        >
+          6
+        </button>
+        <button
+          type="submit"
+          className="side"
+          onClick={(e) => {
+            handleClick(e);
+          }}
+        >
+          -
+        </button>
+        <button
+          type="submit"
+          onClick={(e) => {
+            handleClick(e);
+          }}
+        >
+          1
+        </button>
+        <button
+          type="submit"
+          onClick={(e) => {
+            handleClick(e);
+          }}
+        >
+          2
+        </button>
+        <button
+          type="submit"
+          onClick={(e) => {
+            handleClick(e);
+          }}
+        >
+          3
+        </button>
+        <button
+          type="submit"
+          className="side"
+          onClick={(e) => {
+            handleClick(e);
+          }}
+        >
+          +
+        </button>
+        <button
+          type="submit"
+          className="span"
+          onClick={(e) => {
+            handleClick(e);
+          }}
+        >
+          0
+        </button>
+        <button
+          type="submit"
+          onClick={(e) => {
+            handleClick(e);
+          }}
+        >
+          .
+        </button>
+        <button
+          type="submit"
+          className="side"
+          onClick={(e) => {
+            handleClick(e);
+          }}
+        >
+          =
+        </button>
+      </div>
+    </div>
+  );
+}
+export default CalculatorUI;
```

Comment:
- [OPTIONAL] I would suggest you show each number and the sign on the calculator it would be way better for user experience for example if I could see 5 + 2 it would be way better than currently is